### PR TITLE
Fuzzer updates

### PR DESCRIPTION
**Pull request by kroeckx**

### CODE REVIEW

File: fuzz/server.c [hunk 1]
```diff
@@ -189,15 +190,58 @@ static const uint8_t kRSAPrivateKeyDER[] = {
...
+    BIO *out;
+    SSL_CTX *ctx;
+    int ret;
     RSA *privkey;
+    const uint8_t *bufp = kRSAPrivateKeyDER;
     EVP_PKEY *pkey;
-    int ret;
     X509 *cert;
 
+    if (!len) {
```
Comment (richsalz) [inline fuzz/server.c]: extra curlies.  (admit it, this was just a test to see if anyone else read past all the binary diffs :)

Comment (kroeckx) [inline fuzz/server.c]: And I thought I cleaned up all the style issues ...

Comment (kroeckx) [inline fuzz/server.c]: I guess I missed one, it's actually code moved from below.

**Comment by kroeckx** [comment]
So I pushed a new commit for that.

**Comment by richsalz** [comment]
+1

**Pull request closed without merging**